src/www/ui/api/documentation/openapi.yaml

```diff
@@ -1615,6 +1615,61 @@ paths:
                 $ref: '#/components/schemas/Info'
         default:
           $ref: '#/components/responses/defaultResponse'
+  /uploads/{id}/item/{itemId}/copyrights/{hash}:
+    parameters:
+      - name: id
+        required: true
+        description: upload ID
+        in: path
+        schema:
+          type: integer
+      - name: itemId
+        required: true
+        description: Upload tree ID
+        in: path
+        schema:
+          type: integer
+      - name: hash
+        required: true
+        description: copyright hash
+        in: path
+        schema:
+          type: string
+    put:
+      operationId: updateFileCopyrights
+      tags:
+        - Copyright
+      summary: Updates a copyright for a file
+      description: Updates a copyright statement for a particular file
+      requestBody:
+        description: ClearingInfo payload
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SetCopyrightInfo'
+      responses:
+        '200':
+          description: OK
+          headers:
+            X-Total-Pages:
+              description: Total number of pages which can be generated based on limit
+              schema:
+                type: integer
```

Comment:
The endpoint is not generating this header in the code and is not needed as well.